app/main.py

```diff
@@ -2,4 +2,64 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list[int] = None
+                 ) -> None:
+        self.name = name
+        self.coords = coords or [0, 0]
+        self.weight = weight
+
+    def go_forward(self, step: int = 1) -> None:
+        self.coords[1] += step
+
+    def go_back(self, step: int = 1) -> None:
+        self.coords[1] -= step
+
+    def go_right(self, step: int = 1) -> None:
+        self.coords[0] += step
+
+    def go_left(self, step: int = 1) -> None:
+        self.coords[0] -= step
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list[int] = None
+                 ) -> None:
+        self.coords = coords or [0, 0, 0]
+        super().__init__(name=name, weight=weight, coords=self.coords)
```

Comment:
```suggestion
       coords = coords or [0, 0, 0]
        super().__init__(name=name, weight=weight, coords=coords)
```
Better to do like this